tests/testthat/test-pip.R

```diff
@@ -111,14 +111,17 @@ test_that("year selection is working", {
   expect_equal(tmp$reporting_year, check)
```

Comment:
I cannot comment on the actual line num 110 but 

```
check <- max(lkup$ref_lkup$reporting_year)
```

takes `max` of entire `ref_lkup` which consists of all the countries. Shouldn't we take `max` of only the country passed (in this case `"AGO"`) ? So it should be something like - 

```
check <- max(lkup$ref_lkup$reporting_year[lkup$ref_lkup$country_code == "AGO"])
```

Reply:
Yes, I thought I had removed this...